app/pubspec.yaml

```diff
@@ -32,3 +32,9 @@ dev_dependencies:
   build_runner: ^0.4.0
   source_gen: '^0.7.0'
   test: ">=0.12.20 <0.13.0"
+
+dependency_overrides:
+  appengine:
+    git:
+      url: https://github.com/dart-lang/appengine
+      ref: memcache_fun
```

Comment:
Please don't make PRs with things that should not be committed!

Reply:
See WIP in the title – hoping folks pay attention before hitting merge – it was an easy way to share and discuss the change.